e-commerce-website/src/pages/Auth/Register/index.jsx

```diff
@@ -1,7 +1,85 @@
 import React from 'react'
+import { Link } from 'react-router-dom'
+import { path } from 'src/constants/path'
+import '../AuthSection/styles.scss'
+import { Formik, Form } from 'formik'
+import InputField from 'src/components/InputField'
+import * as Yup from 'yup'
+import PasswordField from 'src/components/PasswordField'
+import AuthSection from '../AuthSection'
 
 function Register(props) {
-  return <div>Register</div>
+  const initialValues = {
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: '',
+    confirmPassword: ''
+  }
+
+  const validationSchema = Yup.object().shape({
+    firstName: Yup.string()
+      .max(15, 'Vui lòng nhập tối đa 15 kí tự trở xuống')
+      .required('Tên là bắt buộc nhập'),
+    lastName: Yup.string()
+      .max(15, 'Vui lòng nhập tối đa 15 kí tự trở xuống')
+      .required('Họ là bắt buộc nhập'),
+    email: Yup.string()
+      .min(6, 'Email có độ dài từ 6 - 160 kí tự')
+      .max(160, 'Email có độ dài từ 6 - 160 kí tự')
+      .email('Email này không hợp lệ')
+      .required('Email là bắt buộc nhập'),
+    password: Yup.string()
+      .min(6, 'Mật khẩu có độ dài từ 6 - 160 kí tự')
+      .max(160, 'Mật khẩu có độ dài từ 6 - 160 kí tự')
+      .required('Mật khẩu là bắt buộc nhập'),
+    confirmPassword: Yup.string()
+      .oneOf([Yup.ref('password'), null], 'Nhập lại mật khẩu chưa chính xác')
```

Comment:
```suggestion
      .oneOf([Yup.ref('password'), null], 'Mật khẩu xác nhận không chính xác')
```
Tiếng Việt kì thế :v

Reply:
> Tiếng Việt kì thế :v

:v lúc đầu em cũng tính khi không nhưng mà sau nghĩ hiện tại họ nhập chưa đúng chứ tương lai họ nhập đúng nên em sửa không thành chưa :v. Mà em thấy nghĩa cũng same same